gnome/gnome-calculator/Portfile

```diff
@@ -1,14 +1,15 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           meson 1.0
 PortGroup           compiler_blacklist_versions 1.0
 PortGroup           yelp 1.0
 
 name                gnome-calculator
-version             3.26.0
+version             46.1
 revision            1
```

Comment:
the "revision" field should be reset to "0" when updating the bersion

Reply:
Whoops, fixed